### PR TITLE
Creating basic Hello World app.

### LINKED ISSUE #1: Summer 2021 Practice
We need to create a front end web application that presents products for an online store. The application can be in any language/framework desired (Java/Spring, JS/Angular, or Python/Flask are good choices).

### PR DESCRIPTION
**Pull request by CaiN79**
Get starting application up and running.
Using Spring framework to print greeting on screen.

#1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
```
Comment (CaiN79) [inline pom.xml:34]: Comment

File: src/main/java/com/nathancai/nfcu/learningspring/Greeting.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.nathancai.nfcu.learningspring;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Greeting {
+
+	public static void main(String[] args) {
+		SpringApplication.run(Greeting.class, args);
```
Comment (Dirkadin) [inline src/main/java/com/nathancai/nfcu/learningspring/Greeting.java:10]: eww tabs

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (Dirkadin) [inline src/main/resources/application.properties:1]: I'm okay with keeping this, but generally we need to remove empty files.

**Review by Dirkadin — APPROVED** [review]

File: src/test/java/com/nathancai/nfcu/learningspring/LearningSpringApplicationTests.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package com.nathancai.nfcu.learningspring;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class LearningSpringApplicationTests {
+
+	@Test
+	void contextLoads() {
```
Comment (tab-dev) [inline src/test/java/com/nathancai/nfcu/learningspring/LearningSpringApplicationTests.java:10]: We typically have tests validate our code. But I'll allow this once :)

**Review by tab-dev — APPROVED** [review]
Other than the no tests thing, we're good

**Pull request merged**